packages/valory/skills/termination_abci/behaviours.py

```diff
@@ -500,7 +500,7 @@ def callback_request(
             :return: none
             """
             if self.is_stopped:
-                self.context.logger.debug(
+                self.context.logger.info(
```

Comment:
there is an [open issue on open-aea](https://github.com/valory-xyz/open-aea/issues/479) for this